tachyon/crypto/commitments/pedersen/pedersen.h

```diff
@@ -118,7 +118,8 @@ class Pedersen final
       if constexpr (std::is_same_v<Commitment, Bucket>) {
         batch_commitments = std::move(cpu_batch_commitments_);
       } else {
-        batch_commitments.resize(cpu_batch_commitments_.size());
+        batch_commitments.resize(cpu_batch_commitments_.size(),
```

Comment:
It seems like we don't need to initialize the values, since `BatchNormalize` will be filled with the values. But just add a comment that why this is safe.